doc/releases/changelog-dev.md

```diff
@@ -55,6 +55,16 @@
   [(#1357)](https://github.com/PennyLaneAI/catalyst/pull/1357)
   [(#1385)](https://github.com/PennyLaneAI/catalyst/pull/1385)
 
+* A new peephole optimization pass, `--disentangle-CNOT`, is available.
```

Comment:
I don't know if this can be considered a peephole optimization, since it relies on a global analysis

Reply:
Good point! The state machine is certainly global. Removed (https://github.com/PennyLaneAI/catalyst/pull/1154/commits/446bf30831a90781625395b33f3205374e0b2a82)